jobs/build/rhcos/Jenkinsfile

```
@@ -47,57 +47,86 @@ node {
             'ppc64le': 'jenkins_serviceaccount_ocp-ppc.stage.psi.redhat.com',
             's390x': 'jenkins_serviceaccount_osbs-s390x-3.prod.engineering.redhat.com.kubeconfig',
             'aarch64': 'jenkins_serviceaccount_osbs-aarch64-1.engineering.redhat.com',
+            'multi': 'multi_jenkins_serviceaccount_ocp-virt.prod.psi.redhat.com.kubeconfig',
         ]
 
         // Disabling compose lock for now. Ideally we achieve a stable repo for RHCOS builds in the future,
         // but for now, being this strict is slowing down the delivery of nightlies.
         //lock("compose-lock-${params.BUILD_VERSION}") {
         lock(resource: "rhcos-lock-${params.BUILD_VERSION}", skipIfLocked: true) {  // wait for all to succeed or fail for this version before starting more
-        timestamps {
-            def archJobs = [:]
-            for (arch in arches) {
-                def jobArch = arch.trim() // make sure we use a locally scoped variable
-                if (!releaseArches.contains(jobArch)) {
-                    echo "Skipping ${jobArch} since ${params.BUILD_VERSION} only supports ${releaseArches}"
-                    continue
+
+        // Check if urls.rhcos_release_base.multi is defined in group.yml
+        def cmd = "doozer -g openshift-${params.BUILD_VERSION} config:read-group urls.rhcos_release_base.multi"
+        multi_builds_enabled = commonlib.shell(script: cmd, returnStatus: true)
+
+        if (multi_builds_enabled == 0) {
```

Then this can be changed to
```suggestion
        if (!multi_builds_enabled) {
```